ietf/doc/templatetags/ietf_filters.py

```diff
@@ -228,6 +229,26 @@ def link_other_doc_match(match):
     return f'<a href="{url}">{match[1]}</a>'
 
 
+@register.filter(name="markdown", is_safe=True, needs_autoescape=True)
+def markdown(string, autoescape=None):
+    """
+    Render a markdown string into HTML
+    """
+    string = bleach_cleaner.clean(      
+        python_markdown.markdown(
+            string,
+            extensions=[
+                "extra",
+                "nl2br",
+                "sane_lists",
+                "toc",
+            ],
+        )
+    )
```

Comment:
Why don't you call the code in ietf/utils/markdown.py?

Reply:
Because of a loop between the import statement (and no clue how to break this loop)

Reply:
In the same vein, I am unsure whether "toc" & "nl2br" extensions are useful or dangerous

Reply:
https://stackoverflow.com/a/7199514/2240756